src/object/wind.cpp

```diff
@@ -69,8 +75,9 @@ Wind::get_settings()
   result.add_float(_("Acceleration"), &acceleration, "acceleration");
   result.add_bool(_("Blowing"), &blowing, "blowing", true);
   result.add_bool(_("Affects Badguys"), &affects_badguys, "affects-badguys", false);
+  result.add_bool(_("Fancy Particles"), &fancy_wind, "fancy-wind", false);
```

Comment:
Not sure about what others might think, but I consider the name "Fancy Particles" to not be verbose enough.